tutorials/intro/Intro_Tutorial_2.ipynb

```diff
@@ -279,8 +279,70 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### Traversing the _Context Hierarchy_\n",
+    "\n",
+    "As you have already probably observed, in Snorkel, all `Candidate`s are basically just tuples of `Context`-type objects--in this (and most) cases, `Span`s. Given a `Candidate`, we can easily access its `Context`s:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "c = train_cands[0]\n",
+    "c.get_contexts()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These `Context` objects are part of a hierarchy of `Context` objects in Snorkel.  In our case, this hierarchy consists of `Document`s, `Sentence`s, and `Span`s.  We can traverse this hierarchy by using the specific names--e.g., `doc.sentences`--or by using the generic `get_parent()` and `get_children()` methods:"
```

Comment:
This should probably be merged with and come before the explanation of `Candidate`s being composed of `Context`s